driver-core/src/main/java/com/datastax/driver/core/Statement.java

```diff
@@ -431,6 +431,9 @@ public Statement setIdempotent(boolean idempotent) {
      * {@link com.datastax.driver.core.querybuilder.QueryBuilder#raw(String)} anywhere in an inserted value,
      * the result will be {@code false}; otherwise it will be {@code true}.
      * In all cases, calling {@link #setIdempotent(boolean)} forces a value that overrides every other mechanism.
+     * <p>
+     * Idempotence configured in a Statement instance will be propagated to the PreparedStatements created from this
```

Comment:
Suggestion: "`Note that when a statement {@link Session#prepare(String) is prepared}, its idempotence flag will be propagated to all {@link PreparedStatement}s created from it.`"